src/components/CreateClient/form.js

```diff
@@ -0,0 +1,105 @@
+import React from 'react';
+import { PropTypes as PT } from 'prop-types';
+
+import { FormContainer } from './styled';
+import { FormStyleContainer } from '../common_styled/FormStyleContainer';
+import { withFormik } from 'formik';
+
+
+const FormikEnhancer = withFormik({
+
+  displayName: 'Create Client Form',
+
+  // This updates the form when props change.
+  enableReinitialize: true,
+
+  // validateOn
+  mapPropsToValues: () => {
+    return ({
+      clientName: '',
+    });
+  },
+
+  // Custom sync validation
+  validate: ({ clientName }) => {
+    let errors = {};
+    if (clientName === '') {
```

Comment:
is this the only validation we need?- nothing about numbers or max length etc?

Reply:
Going by what existed before implementing formik, this was the only validation required.